src/diffusers/modeling_utils.py

```diff
@@ -689,3 +642,101 @@ def unwrap_model(model: torch.nn.Module) -> torch.nn.Module:
         return unwrap_model(model.module)
     else:
         return model
+
+
+def get_model_file(pretrained_model_name_or_path, **kwargs) -> Tuple[str, bool]:
```

Comment:
The functions become a bit too nested for me here and hard to follow - could we maybe just remove this function and only keep `_get_model_file`? I don't think those 8 lines deserve a new function here, I'd prefer to just call `_get_model_file` directly above.

Reply:
Happy to remove it, but I think the calling function will get messier.